cpu/kinetis/Makefile.features

```diff
@@ -1,5 +1,18 @@
 FEATURES_PROVIDED += periph_cpuid
-FEATURES_PROVIDED += periph_hwrng
+
+# HACK Do not define 'hwrng' if the board does not supports it
+# I prefer a whitelist on CPU_MODEL but the information is not available yet
```

Comment:
This comment has too much personal taste (e.g. don't use  `I prefer`). Maybe reword with: `A whitelist on CPU_MODEL would be better but this information/variable is not available yet.`

Reply:
Indeed, will change.